osu.Game.Rulesets.Osu/Skinning/Default/ManualSliderBody.cs

```diff
@@ -11,10 +11,17 @@ namespace osu.Game.Rulesets.Osu.Skinning.Default
     /// </summary>
     public partial class ManualSliderBody : SliderBody
     {
-        public new void SetVertices(IReadOnlyList<Vector2> vertices)
+        public new void SetVertices(IReadOnlyList<Vector2> vertices) => base.SetVertices(vertices);
```

Comment:
This shadowed method has no reason to even exist anymore if it's just gonna call base.

Reply:
It's required in this case because the `base` method is `protected` :/